src/partials/comments.html

```diff
@@ -0,0 +1,103 @@
+<div class="comments container">
+    <div class="comments__view">
+        <div class="comments__card-home" id="card-home" >
+            <img class="comments__image" src="/images/comments/comment-photo1.jpg" alt="zdjecie portretowe Emily, Los Angeles" width="85" height="85"
+            srcset="/images/comments/comment-photo1@2x.jpg 2x">
+            <div class="comments__sheet">
+                <svg class="comments__sheet-icon" width="54" height="40">
+                    <use href="/images/sprite/icons.svg#icon-quote"></use>
+                </svg>
+                <div class="comments__line"></div>
+                <div class="comments__line"></div>
+                <div class="comments__line invisible-desktop"></div>
+                <div class="comments__line invisible-tablet-desktop"></div>
+                <div class="comments__line invisible-tablet-desktop"></div>
+                <div class="comments__line invisible-tablet-desktop"></div>
+                <div class="comments__line invisible-tablet-desktop"></div>
+                <div class="comments__line invisible-tablet-desktop"></div>
+                <p class="comments__content">Nullam viverra consectetuer. Quisque cursus et, porttitor risus. Aliquam sem. In hendrerit nulla quam nunc, accumsan congue. Lorem ipsum primis in nibh vel risus.</p>
+            </div>
+            <ul class="comments__dots-list link">
+                <li class="comments__dot"></li>
+                <li class="comments__dot"></li>
+                <li class="comments__dot"></li>
+            </ul>
+            <p class="comments__author">Emily, Los Angeles</p>
+            <ul class="comments__navigation">
+                <li class="comments__post-item"><a href="#card-home" class="comments__post-link">
+                    <svg class="comments__navigation-icon" width="12.4" height="11.84">
+                    <use href="/images/sprite/icons.svg#icon-home"></use>
+                    </svg>
+                </a></li>
+                <li class="comments__post-item"><a href="#card-second" class="comments__post-link comments__post-link--bg-styles"></a></li>
+                <li class="comments__post-item"><a href="#card-third" class="comments__post-link comments__post-link--bg-styles"></a></li>
+            </ul>
+        </div>
+        <div class="comments__card-second" id="card-second">
+            <img class="comments__image" src="/images/comments/comment-photo2.jpg" alt="zdjecie portretowe Emily, Los Angeles" width="85" height="85"
+            srcset="/images/comments/comment-photo2@2x.jpg 2x">
```

Comment:
Czy w Ciebie podrzuca inne zdjęcie przy ekranie Retina przy takim kodzie?

Ja wykorzystuję w taki sposób:
                    srcset="./images/logo/logo.png  200w,
                    ./images/logo/logo2x.png  400w,
                    "

Czyli powtarzam to co jest w src oraz zdjęcie 2x w srcset

Reply:
Tak jak wybierzesz zagęszczenie ekranu obok wysokości na 2.0 w dev tools(f12) to kiedy najedziesz na sciezke w tym podglądzie wyswietli ci sie aktualnie ładowane zdjęcie (sciezka